<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Test" duration-ms="64447" started-at="2017-02-27T19:23:49Z" finished-at="2017-02-27T19:24:54Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="64447" started-at="2017-02-27T19:23:49Z" finished-at="2017-02-27T19:24:54Z">
      <class name="chenjian.tests.AddNews2">
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="4" started-at="2017-02-27T19:23:49Z" finished-at="2017-02-27T19:23:49Z">
        </test-method>
        <test-method status="FAIL" signature="addOne2(java.util.Map)" name="addOne2" duration-ms="20260" started-at="2017-02-27T19:24:32Z" finished-at="2017-02-27T19:24:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{content=chenjian444, title=chenjian3, imgSrc=E:\\win8常见问题\\qqq.jpg, browser=firefox}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[元素获取异常,值为空!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: 元素获取异常,值为空!
	at systemPublic.utilities.YamlParser.getDynamicElement(YamlParser.java:99)
	at chenjian.tests.AddNews2.addOne2(AddNews2.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:643)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:559)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1027)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1030)
	at org.testng.TestRunner.privateRun(TestRunner.java:709)
	at org.testng.TestRunner.run(TestRunner.java:579)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:331)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:326)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:288)
	at org.testng.SuiteRunner.run(SuiteRunner.java:193)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:910)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:93)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:145)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:87)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="teardown()" name="teardown" is-config="true" duration-ms="1996" started-at="2017-02-27T19:24:52Z" finished-at="2017-02-27T19:24:54Z">
        </test-method>
      </class>
      <class name="chenjian.tests.AddNews">
        <test-method status="FAIL" signature="addOne(java.util.Map)" name="addOne" duration-ms="22315" started-at="2017-02-27T19:23:49Z" finished-at="2017-02-27T19:24:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{content=chenjian233, title=chenjian2, imgSrc=E:\\win8常见问题\\qqq.jpg, browser=firefox}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[元素获取异常,值为空!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: 元素获取异常,值为空!
	at systemPublic.utilities.YamlParser.getDynamicElement(YamlParser.java:99)
	at chenjian.tests.AddNews.addOne(AddNews.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:643)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:559)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1027)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1030)
	at org.testng.TestRunner.privateRun(TestRunner.java:709)
	at org.testng.TestRunner.run(TestRunner.java:579)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:331)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:326)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:288)
	at org.testng.SuiteRunner.run(SuiteRunner.java:193)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:910)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:93)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:145)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:87)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="FAIL" signature="addOne2(java.util.Map)" name="addOne2" duration-ms="19856" started-at="2017-02-27T19:24:12Z" finished-at="2017-02-27T19:24:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{content=chenjian444, title=chenjian3, imgSrc=E:\\win8常见问题\\qqq.jpg, browser=firefox}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[元素获取异常,值为空!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: 元素获取异常,值为空!
	at systemPublic.utilities.YamlParser.getDynamicElement(YamlParser.java:99)
	at chenjian.tests.AddNews.addOne2(AddNews.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:643)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:559)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1027)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1030)
	at org.testng.TestRunner.privateRun(TestRunner.java:709)
	at org.testng.TestRunner.run(TestRunner.java:579)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:331)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:326)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:288)
	at org.testng.SuiteRunner.run(SuiteRunner.java:193)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:910)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:93)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:145)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:87)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
